README.md

```diff
@@ -26,6 +26,8 @@ The Stackable HDFS already takes care of this, you don't need to do anything in
 
 - Set `dfs.namenode.inode.attributes.provider.class` in `hdfs-site.xml` to `tech.stackable.hadoop.StackableAuthorizer`
 - Set `hadoop.security.authorization.opa.policy.url` in `core-site.xml` to the HTTP endpoint of your OPA rego rule, e.g. `http://opa.default.svc.cluster.local:8081/v1/data/hdfs/allow`
+- The property `hadoop.security.authorization.opa.extended-requests` (defaults to `false`), which controls if all fields should be send to OPA.
```

Comment:
```suggestion
- The property `hadoop.security.authorization.opa.extended-requests` (defaults to `false`), which controls if all fields should be sent to OPA.
```

Reply:
This needs documentation somewhere which fields it will always send.

Reply:
Please have a look at e2124091afadeab7a22f7f5ab155e1628fce0779